webrtc.html

```diff
@@ -3739,6 +3769,25 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       </section>
 
       <section>
+        <h3>RTCRtpTransceiver Interface</h3>
+
+        <dl class="idl" title="dictionary RTCRtpTransceiverInit">
+          <dt>sequence&lt;RTCRtpEncodingParameters&gt; sendEncodings</dt>
```

Comment:
The problem is that you can't getParameters() until after a remote description has been set.  So to set the encodings before calling createOffer, you'd have to call setParameters with a partial set of parameters (no codecs, header extensions, etc), which would leave funny questions about which things make sense in setParameters at which times.  

This seemed more simple.